app/views/shared/_modal_layout.slim

```diff
@@ -0,0 +1,5 @@
+.modal-cntnr
```

Comment:
Now that we only have to update this once, what do you think about changing the class name to `modal-backdrop` or something similar?